lib/modules/hosts/gatherer.js

```diff
@@ -0,0 +1,15 @@
+export default {
+	moduleID: 'gatherer',
+	name: 'gatherer',
```

Comment:
`name` can be more descriptive than the `moduleID`, maybe something like `'MTG Card Images'` or `'Magic TCG Cards'`